.travis.yml

```diff
@@ -3,21 +3,26 @@ language: java
 jdk:
   - oraclejdk8
 
-script: "./travis.sh"
+before_cache:
+- rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
+- rm -fr $HOME/.gradle/caches/*/plugin-resolution/
+- rm -fr $HOME/.gradle/caches/*/fileHashes/fileHashes.bin
+- rm -fr $HOME/.gradle/caches/*/fileHashes/fileHashes.lock
+
 cache:
   directories:
-    - .cache
-before_script:
- - sudo apt-get -qq update
- - sudo apt-get install ant-optional
- - chmod +x travis.sh
+  - $HOME/.gradle/caches/
+  - $HOME/.gradle/wrapper/
+
+before_install:
+- "export ORG_GRADLE_PROJECT_ideaVersion=${IDEA_VERSION}"
+- "export ORG_GRADLE_PROJECT_phpPluginVersion=${PHP_PLUGIN_VERSION}"
+- "export ORG_GRADLE_PROJECT_twigPluginVersion=${TWIG_PLUGIN_VERSION}"
+- java -version
```

Comment:
This is useless. Travis already displays the java version. No need to duplicate it in a collapsed block: https://travis-ci.org/Haehnchen/idea-php-toolbox/jobs/354292097#L463 vs https://travis-ci.org/Haehnchen/idea-php-toolbox/jobs/354292097#L477

Reply:
Removed